ovos_padatious/opm.py

```
@@ -13,26 +13,97 @@
+    def __init__(self, lang):
+        lang2 = closest_match(lang, list(self.LANGS))[0]
+        if lang2 == "und":
+            raise ValueError(f"unsupported language: {lang}")
+        self.snowball = snowballstemmer.stemmer(self.LANGS[lang2])
+
+    @classmethod
+    def supports_lang(cls, lang) -> bool:
+        lang2 = closest_match(lang, list(cls.LANGS))[0]
+        return lang2 != "und"
+
+    @lru_cache()
```

Ruff (0.8.2)

Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)